### ISSUE TITLE
khard does not understand Apple group files

### ISSUE THREAD

**Issue by 0-wiz-0** [scheibler/khard#88]
I sync my contacts via vdirsyncer. A couple of the files that are synced this way are contact group files. khard displays them like standard contacts.

One example looks like this (slightly edited):
````
BEGIN:VCARD
VERSION:3.0
PRODID:-//Apple Inc.//AddressBook 9.0//EN
N:My Group Name;;;;
FN:My Group Name
X-ADDRESSBOOKSERVER-KIND:group
X-ADDRESSBOOKSERVER-MEMBER:urn:uuid:$UID_OF_CONTACT1-ABSPlugin
X-ADDRESSBOOKSERVER-MEMBER:urn:uuid:$UID_OF_CONTACT2-ABSPlugin
...
X-RADICALE-NAME:$SOME_UID-ABSPlugin.vcf
REV:2016-04-10T12:15:44Z
UID:$UID_OF_GROUP
END:VCARD
````
(I guess the X-RADICALE-NAME got added because I was using radicale at some point and can be ignored.)

It would be good if khard understood these files correctly and could perhaps even edit them.

One particular request in that direction: show me all contacts that are in no group.

**Comment by scheibler** [comment]
Indeed, khard could support the [kind](https://tools.ietf.org/html/rfc6350#section-6.1.4) and
[member](https://tools.ietf.org/html/rfc6350#section-6.6.5) attributes of a vcard. For example I
could add an option that only shows individuals and filters group vcards. I'll add that to my todo
list.

But the problem is, that Apple had to define their own private objects (x-) or Apple invented that
and it was included into the standard afterwards. Anyway, your vcards still wouldn't work cause
khard classifies all attributes with a leading x- as private (see example readme file
misc/khard/khard.conf.example).

For the moment you also could sort all group cards into a single address book and leave that out by
selecting all remaining address books with -a.

**Comment by 0-wiz-0** [comment]
Support for `kind` and `member` would be great, even if it doesn't solve my current problem.
I understand why you wouldn't want to support undocumented Apple extensions, so feel free to close this.

**Comment by brendan-r** [comment]
Sabre/Kolab does a similar thing (in my case, also passed through [CardBook](https://addons.mozilla.org/en-US/thunderbird/addon/cardbook/))

```
BEGIN:VCARD
VERSION:4.0
PRODID:-//Kolab//iRony DAV Server 0.3.1//Sabre//Sabre VObject 2.1.7//EN
UID:ebc93358-69bd-11e6-b6b5-52549cdcb91e
FN:Test
REV:20160824T055322Z
KIND:group
MEMBER:urn:uuid:CF5E7E1C-E6D8C038-79B7DEFA
MEMBER:urn:uuid:03a0578a-69bf-11e6-aa1c-52549cdcb91e
X-THUNDERBIRD-ETAG:"5f9d7dc5ddb7d5b4-2640"
END:VCARD
```
